old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/07-spatial-resolution-extent.Rmd

```diff
@@ -65,95 +95,144 @@ directory to the location of the downloaded and unzipped data subsets.
 in order to effectively work with spatial data:
 
-* **Coordinate Reference System** (CRS),
+* **Coordinate Reference System** (CRS)
 * **Extent**
 * **Resolution**
 
-
+## Coordinate Reference System
+The Coordinate Reference System, or CRS, tells R where the raster is located in 
+geographic space. It also tells R what method should be used to "flatten" or 
+project the raster in geographic space. 
+ 
+It is important when working with spatial data in a program like R to identify 
+the coordinate reference system applied to the data and retain it throughout data 
+processing and analysis.  Otherwise, data from the same location but saved in 
+different projections will not line up in any GIS or other program.
 
 ## Spatial Extent
-The spatial extent of a spatial object is just how much area does it cover. A
-map of Paris has a smaller spatial extent than a map of all of France.  
+The spatial extent of a spatial object is simply how much area does it cover. For 
+example, a map of Paris has a smaller spatial extent than a map of all of France.  
 
 ### Units
 The units of the extent are defined by the coordinate system that the spatial
-data is in.  
+data is in. A UTM coordinate system is in meters, for example. Geographic coordinate 
+reference systems are in latitude and longitude, which is most often recorded in 
+decimal degrees.
 
 ### Extent in Vector Data
-GRAPHIC FROM COLIN
+<figure>
+    <a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/vector-general/spatial_extent.png">
+    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/vector-general/spatial_extent.png" 
+    alt=”The three types of vectors (points, lines, and polygons) are represented 
+    within the given (8,8) spatial extent.”></a>
+    <figcaption>The spatial extent of a shapefile or R spatial object represents 
+    the geographic "edge" or location that is the furthest north, south, east, 
+    and west. Thus it represents the overall geographic coverage of the spatial object.
+    </figcaption>
+</figure>
+
+The spatial extent of a vector dataset is determined by the min and max “X,Y” 
+location of **all** the spatial objects stored in the dataset. The spatial extent 
+of an **individual** object can also be quantified!
+
+Let’s look at the spatial extent of a line shapefile representing roads at the <a href="https://www.neonscience.org/field-sites/field-sites-map/HARV" target="_blank"> NEON Harvard Forest field site</a>.
+
+``` {r vector-extent}
+#load libraries
+library(rgdal)
+library(raster)
+
+# set working directory to ensure R can find the file we wish to import and where 
+# we want to save our files. Be sure to move the unzipped download into your working directory! 
+wd <- "~/Documents/data/" #This will depend on your local environment 
+setwd(wd)
+
+# import a line shapefile
+lines_HARV <- readOGR(paste0(wd,"NEON-DS-Site-Layout-Files/HARV"), "HARV_roads")
+
```

Comment:
What do you think about plotting each of these files to demonstrate what they 'look' like before referring to their CRS and extent?

Reply:
Good idea! I can definitely work on adding that.

Reply:
@donal-at-NEON - all my updates are in for this tutorial. I wanted to call your attention to what I did for this vector plot. I used `ggplot()` instead of `plot()` so we could see the lat and long units on the axes. I think this is the best way to really show the extent on a plot to get a nice visual (versus using `plot()` that doesn't automatically include axes).

Reply:
Great! `ggplot()` almost always looks much nicer than `plot()`